packages/core/src/fetchers/createExperience.ts

```diff
@@ -1,15 +1,13 @@
 import type { Asset, Entry } from 'contentful';
 import { isExperienceEntry } from '@/utils';
-import type { Experience, ExternalSDKMode } from '@/types';
+import type { Experience } from '@/types';
 import { EntityStore } from '@/entity/EntityStore';
 
 type createExperienceArgs = {
   experienceEntry: Entry;
   referencedEntries: Array<Entry>;
   referencedAssets: Array<Asset>;
   locale: string;
-  /** @deprecated mode no longer needed */
-  mode?: ExternalSDKMode;
```

Comment:
Is createExperienceArgs used in existing customer apps?

Reply:
Technically likely yes.  But since mode wasn't actually used anywhere in the codebase, it was just a deprecated arg this wouldn't introduce any breaking changes.